Tema-1/script4.js

```diff
@@ -0,0 +1,10 @@
+console.log("EXERCITIUL 4")
+
+let nume;
+nume=Ghertan
```

Comment:
aici Ghertan se refera la variabila numita Ghertan care nu exista in loc de textul "Ghertan"